handlers/incident.go

```diff
@@ -33,6 +36,70 @@ func apiServiceIncidentsHandler(w http.ResponseWriter, r *http.Request) {
 	returnJson(service.Incidents, w, r)
 }
 
+func apiServiceIncidentsHandlerActive(w http.ResponseWriter, r *http.Request) {
```

Comment:
Is `apiServiceIncidentsHandlerActive` as per the naming convention in this repo.
It is not very descriptive.